lumigator/backend/backend/config_templates.py

```diff
@@ -150,13 +149,14 @@
     }}
 }}"""
 
-
 templates = {
     JobType.INFERENCE: {
         "default": default_infer_template,
-        "oai://gpt-4o-mini": oai_infer_template,
-        "oai://gpt-4o": oai_eval_template,
-        "mistral://open-mistral-7b": oai_infer_template,
+        "gpt-4o-mini": oai_infer_template,
+        "gpt-4o": oai_infer_template,
+        "mistral/open-mistral-7b": oai_infer_template,
+        "deepseek/deepseek-reasoner": oai_infer_template,
+        "deepseek/deepseek-chat": oai_infer_template,
```

Comment:
Note that I didn't yet update the models.yaml file, because the frontend doesn't use this logic, and the frontend doesn't yet support LiteLLM. this means that the "get_suggested_models()" route won't work for the SDK, because that route is going to give back the model list of models.yaml, which is only applicable for JobType.EVALUATION, not JobType.INFERENCE.